python/pyarrow/table.pxi

```diff
@@ -1058,12 +1058,12 @@ cdef class Table:
     @staticmethod
     def from_batches(batches, Schema schema=None):
         """
-        Construct a Table from a list of Arrow RecordBatches
+        Construct a Table from an iterator of Arrow RecordBatches
```

Comment:
Can you say "a sequence or iterator"? Iterator has a slightly more specific meaning

Reply:
Yeah, that sounds good